spec/rubocop/cop/layout/space_before_brackets_spec.rb

```diff
@@ -48,6 +48,17 @@
       RUBY
     end
 
+    it 'registers an offense and corrects when using space between method argument parentheses and left bracket' do
+      expect_offense(<<~RUBY)
+        collection.call(arg) [inde_or_key]
```

Comment:
I think you meant to write "index" here.

Reply:
Oops! You're right. I've fixed it.